src/qml/components/blockclockdial.cpp

```diff
@@ -15,7 +17,48 @@ BlockClockDial::BlockClockDial(QQuickItem *parent)
 , m_background_color{QColor("#2D2D2D")}
 , m_confirmation_colors{QList<QColor>{}}
 , m_time_tick_color{QColor("#000000")}
+, m_animation_timer{QTimer(this)}
```

Comment:
we still want the delay timer, even though the connection animation grows, we still want the 5 sec delay before it starts.